storybook/pages/ImagesGridViewPage.qml

```diff
@@ -0,0 +1,111 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+import Storybook 1.0
+
+Pane {
+    id: root
+
+    ListModel {
+        id: imagesModel
+    }
+
+    Instantiator {
+        model: 20
+
+        delegate: Rectangle {
+            parent: root
+
+            width: 150
+            height: 150
+
+            color: 'whitesmoke'
+            border.width: 1
+            visible: false
+
+            Label {
+                anchors.centerIn: parent
+                text: "image " + index
+                font.pixelSize: 20
+            }
+
+            Image {
+                id: keepUrlAlive
+                visible: false
+            }
+
+            Component.onCompleted: {
+                imagesModel.append({
+                    imageLink: "",
+                    rawLink: `raw link ${index}`
```

Comment:
How does this work?

Reply:
just generating mock images on the fly from qml components

Reply:
No, I mean the `raw link` part

Reply:
the model expect imageLink (for downloading image directly) and the original (raw) link provided by the user. The second one is used by "link" button to open design in the browser.

Reply:
content of that role is visible when hovering link button